### ISSUE TITLE
Accessibility: ipython doc reference

### ISSUE THREAD

**Issue by Quemmies3** [jupyterhub/outreachy#61]
### What page is this for?

https://ipython.org/ipython-doc/stable/interactive/reference.html

### WAVE accessibility report

Please find link to my comment in #38 

[Wave report from Quemmies3 on IPython Reference page](https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1278366526)

### The HTML element to be changed

```html
**Language missing or invalid:** `<html lang = "">`

**Structural elements: Table of content indentation**
`<h3><a href="../index.html">Table Of Contents</a></h3>
  <ul>
<li><a class="reference internal" href="#">IPython reference</a><ul>
<li><a class="reference internal" href="#command-line-usage">Command-line usage</a><ul>
<li><a class="reference internal" href="#id1">Command-line Options</a></li>
</ul>
</li>
<li><a class="reference internal" href="#interactive-use">Interactive use</a><ul>
<li><a class="reference internal" href="#caution-for-windows-users">Caution for Windows users</a></li>
<li><a class="reference internal" href="#magic-command-system">Magic command system</a></li>
<li><a class="reference internal" href="#access-to-the-standard-python-help">Access to the standard Python help</a></li>
<li><a class="reference internal" href="#dynamic-object-information">Dynamic object information</a></li>
<li><a class="reference internal" href="#readline-based-features">Readline-based features</a><ul>
<li><a class="reference internal" href="#command-line-completion">Command line completion</a></li>
<li><a class="reference internal" href="#search-command-history">Search command history</a></li>
<li><a class="reference internal" href="#autoindent">Autoindent</a></li>
<li><a class="reference internal" href="#customizing-readline-behavior">Customizing readline behavior</a></li>
</ul>
</li>
<li><a class="reference internal" href="#session-logging-and-restoring">Session logging and restoring</a></li>
<li><a class="reference internal" href="#system-shell-access">System shell access</a></li>
<li><a class="reference internal" href="#manual-capture-of-command-output-and-magic-output">Manual capture of command output and magic output</a></li>
<li><a class="reference internal" href="#system-command-aliases">System command aliases</a></li>
<li><a class="reference internal" href="#recursive-reload">Recursive reload</a></li>
<li><a class="reference internal" href="#verbose-and-colored-exception-traceback-printouts">Verbose and colored exception traceback printouts</a></li>
<li><a class="reference internal" href="#input-caching-system">Input caching system</a></li>
<li><a class="reference internal" href="#output-caching-system">Output caching system</a></li>
<li><a class="reference internal" href="#directory-history">Directory history</a></li>
<li><a class="reference internal" href="#automatic-parentheses-and-quotes">Automatic parentheses and quotes</a></li>
</ul>
</li>
<li><a class="reference internal" href="#ipython-as-your-default-python-environment">IPython as your default Python environment</a></li>
<li><a class="reference internal" href="#embedding-ipython">Embedding IPython</a></li>
<li><a class="reference internal" href="#using-the-python-debugger-pdb">Using the Python debugger (pdb)</a><ul>
<li><a class="reference internal" href="#running-entire-programs-via-pdb">Running entire programs via pdb</a></li>
<li><a class="reference internal" href="#post-mortem-debugging">Post-mortem debugging</a></li>
</ul>
</li>
<li><a class="reference internal" href="#pasting-of-code-starting-with-python-or-ipython-prompts">Pasting of code starting with Python or IPython prompts</a></li>
<li><a class="reference internal" href="#gui-event-loop-support">GUI event loop support</a><ul>
<li><a class="reference internal" href="#pyqt-and-pyside">PyQt and PySide</a></li>
</ul>
</li>
<li><a class="reference internal" href="#plotting-with-matplotlib">Plotting with matplotlib</a></li>
<li><a class="reference internal" href="#interactive-demos-with-ipython">Interactive demos with IPython</a></li>
</ul>
</li>
</ul>`
```

### Your proposed HTML (or CSS), after change

```html
**Language missing or invalid:** `<html lang="en">`

**Structural elements: Table of content indentation**
`<h3><a href="../index.html">Table Of Contents</a></h3>
    <ul>
  <li><a class="reference internal" href="#">IPython reference</a><ol>
  <li><a class="reference internal" href="#command-line-usage">Command-line usage</a><ul>
  <li><a class="reference internal" href="#id1">Command-line Options</a></li>
  </ul>
  </li>
  <li><a class="reference internal" href="#interactive-use">Interactive use</a><ul>
  <li><a class="reference internal" href="#caution-for-windows-users">Caution for Windows users</a></li>
  <li><a class="reference internal" href="#magic-command-system">Magic command system</a></li>
  <li><a class="reference internal" href="#access-to-the-standard-python-help">Access to the standard Python help</a></li>
  <li><a class="reference internal" href="#dynamic-object-information">Dynamic object information</a></li>
  <li><a class="reference internal" href="#readline-based-features">Readline-based features</a><ol>
  <li><a class="reference internal" href="#command-line-completion">Command line completion</a></li>
  <li><a class="reference internal" href="#search-command-history">Search command history</a></li>
  <li><a class="reference internal" href="#autoindent">Autoindent</a></li>
  <li><a class="reference internal" href="#customizing-readline-behavior">Customizing readline behavior</a></li>
  </ol>
  </li>
  <li><a class="reference internal" href="#session-logging-and-restoring">Session logging and restoring</a></li>
  <li><a class="reference internal" href="#system-shell-access">System shell access</a></li>
  <li><a class="reference internal" href="#manual-capture-of-command-output-and-magic-output">Manual capture of command output and magic output</a></li>
  <li><a class="reference internal" href="#system-command-aliases">System command aliases</a></li>
  <li><a class="reference internal" href="#recursive-reload">Recursive reload</a></li>
  <li><a class="reference internal" href="#verbose-and-colored-exception-traceback-printouts">Verbose and colored exception traceback printouts</a></li>
  <li><a class="reference internal" href="#input-caching-system">Input caching system</a></li>
  <li><a class="reference internal" href="#output-caching-system">Output caching system</a></li>
  <li><a class="reference internal" href="#directory-history">Directory history</a></li>
  <li><a class="reference internal" href="#automatic-parentheses-and-quotes">Automatic parentheses and quotes</a></li>
  </ul>
  </li>
  <li><a class="reference internal" href="#ipython-as-your-default-python-environment">IPython as your default Python environment</a></li>
  <li><a class="reference internal" href="#embedding-ipython">Embedding IPython</a></li>
  <li><a class="reference internal" href="#using-the-python-debugger-pdb">Using the Python debugger (pdb)</a><ul>
  <li><a class="reference internal" href="#running-entire-programs-via-pdb">Running entire programs via pdb</a></li>
  <li><a class="reference internal" href="#post-mortem-debugging">Post-mortem debugging</a></li>
  </ul>
  </li>
  <li><a class="reference internal" href="#pasting-of-code-starting-with-python-or-ipython-prompts">Pasting of code starting with Python or IPython prompts</a></li>
  <li><a class="reference internal" href="#gui-event-loop-support">GUI event loop support</a><ul>
  <li><a class="reference internal" href="#pyqt-and-pyside">PyQt and PySide</a></li>
  </ul>
  </li>
  <li><a class="reference internal" href="#plotting-with-matplotlib">Plotting with matplotlib</a></li>
  <li><a class="reference internal" href="#interactive-demos-with-ipython">Interactive demos with IPython</a></li>
  </ul>
  </li>
  </ul>`
```

### More info

All redundant links be removed to enhance website performance and include updated information and links.

**Comment by hillaryCyn** [comment]
hi please can you explain better

**Comment by Quemmies3** [comment]
> hi please can you explain better

What do you need clarity on so I can explain better? @hillaryCyn

**Comment by hillaryCyn** [comment]
sorry just wanted to know how we get feed back on the work we have done @Quemmies3

**Comment by minrk** [comment]
Thanks for the suggestions. Can you explain what you mean by "Table of content indentation" and how it relates to accessibility?

**Comment by Quemmies3** [comment]
Thank you Min. I was actually referencing the type of listing used. It caused the table of contents to look not properly indented and confusing as it took a while for me to identify which item was a sublist, making it slightly inaccessible for me as a user.

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
